README.md

```diff
@@ -1,4 +1,4 @@
-ACL (Access Control List)
+acl
```

Comment:
It may be fine since the first time it is used in a sentence in overview it is spelled out.